pokemongo_bot/walkers/step_walker.py

```diff
@@ -72,26 +73,16 @@ def step(self):
             self.bot.heartbeat()
             return True
 
-        totalDLat = (self.destLat - self.initLat)
-        totalDLng = (self.destLng - self.initLng)
-        magnitude = self._pythagorean(totalDLat, totalDLng)
-        unitLat = totalDLat / magnitude
-        unitLng = totalDLng / magnitude
+        self._new_position = self._get_next_pos(self.initLat, self.initLng, self.bearing, self.speed, 1)
```

Comment:
i i think about in for small distances like our speed range... even 10 should do...
we can even make it configurable...